app/views/food/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>New Food</h1>
+
+<%= form_with model: @food, url: food_index_path do |form| %>
+  <%= render "shared/form_errors", form: form %>
+  <div class="mb-3">
+    <%= form.label :name %>
+    <%= form.text_field :name, class:"form-control", placeholder: "name" %>
+  </div>
+  <div class="mb-3">
+    <%= form.label :measurement_unit %>
+    <%= form.text_field :measurement_unit, class:"form-control", placeholder: "measurement unit" %>
+  </div>
+  <div class="mb-3">
+    <%= form.label :price %>
+    <%= form.text_field :price, class:"form-control", placeholder: "price" %>
```

Comment:
It would be better if you use `form.number_field` since the value is a float. You can use `step: 0.05` at the end to admit decimal values.